service/src/main/java/uk/gov/hmcts/reform/fpl/enums/IsAddressKnowType.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.fpl.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.Arrays;
+
+@Getter
+@RequiredArgsConstructor
+public enum IsAddressKnowType {
+    YES("Yes"),
+    NO("No"),
+    LIVE_IN_REFUGEE("LIVE_IN_REFUGE");
```

Comment:
can remove the extra E here on the enum name!

Reply:
Haha, missed that one. Well...still need that fromString for converting "Yes" to IsAddressKnowType.YES.